src/EditorFeatures/Test/CodeFixes/CodeFixServiceTests.cs

```diff
@@ -768,5 +797,132 @@ private void RegisterCodeFix(CodeFixContext context, string titleSuffix)
                     context.Diagnostics);
             }
         }
+
+        [Fact, WorkItem(62877, "https://github.com/dotnet/roslyn/issues/62877")]
+        public async Task TestAdditionalDocumentCodeFixAsync()
+        {
+            var analyzer = new AdditionalFileAnalyzer();
+            var fixer1 = new AdditionalFileFixerWithDocumentKindsAndExtensions();
+            var fixer2 = new AdditionalFileFixerWithDocumentKinds();
+            var fixer3 = new AdditionalFileFixerWithDocumentExtensions();
+            var fixer4 = new AdditionalFileFixerWithoutDocumentKindsAndExtensions();
```

Comment:
Just to understand, the goal of these is they're not included, to confirm the fact we're filtering correctly?

Reply:
Yes, that is correct.